diff --git a/hw/virtio/virtio-pci.c b/hw/virtio/virtio-pci.c
index edbc0da..5a0d562 100644
--- a/hw/virtio/virtio-pci.c
+++ b/hw/virtio/virtio-pci.c
@@ -1906,6 +1906,7 @@ static void virtio_pci_pre_plugged(DeviceState *d, Error **errp)
     VirtIODevice *vdev = virtio_bus_get_device(&proxy->bus);
 
     if (virtio_pci_modern(proxy)) {
+        virtio_add_feature(&vdev->host_features, 0 /* VIRTIO_GPIO_F_IRQ added by WZab to enable interrupts in emulated GPIO */);         
         virtio_add_feature(&vdev->host_features, VIRTIO_F_VERSION_1);
     }
 
